charts/azuremonitor-containers/values.yaml

```diff
@@ -28,7 +28,7 @@ amalogs:
     tagWindows: "win-3.1.15"
     pullPolicy: IfNotPresent
     dockerProviderVersion: "18.0.1-0"
-    agentVersion: "azure-mdsd-1.17.0"
```

Comment:
Will we need this in the aks manifest as well?

Reply:
yes we need to update the aks manifest as well